packages/client/hmi-client/src/components/workflow/ops/calibrate-ensemble-ciemss/tera-calibrate-ensemble-node-ciemss.vue

```diff
@@ -15,7 +15,7 @@
 					selectedRun: props.node.state.forecastRunId,
 					selectedVariable: config
 				}"
-				:mapping="props.node.state.ensembleConfigs as any"
+				:mapping="props.node.state.ensembleMapping as any"
```

Comment:
Does this break the chart now that you have updated the typing here?
This chart is sure to be updated but in the meantime you may need to use the util function you created?

Reply:
added that transform fn here